src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,49 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ELEMENTS_NUMBER = 10;
+    private static final int PAIR = 2;
+    private static final int KEY_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private final Object[][] storage = new Object[MAX_ELEMENTS_NUMBER][PAIR];
+    private int storageSize = 0;
```

Comment:
```suggestion
    private int size;
```

It will be initialized by 0 by default